development-environment-base.json

```diff
@@ -92,8 +105,7 @@
       "iso_checksum_type": "{{user `iso_checksum_type`}}",
       "iso_interface": "sata",
       "iso_url": "{{user `iso_file`}}",
-      "output_directory": "output-{{user `vm_name` }}-{{isotime \"2006-01-02\"}}",
```

Comment:
this line _could_ be a separate change. was meant to be in the clean up change recently